staging/kubeaddons-catalog/templates/crd_addonrepository.yaml

```diff
@@ -1,9 +1,14 @@
+{{- if not (.Capabilities.APIVersions.Has "kubeaddons.mesosphere.io/v1alpha1") }}
```

Comment:
what would happen if `Addon` is already installed? would `.Capabilities.APIVersions` have `"kubeaddons.mesosphere.io/v1alpha1"`?

Reply:
This is part of the problem with how we have catalog as part of Kubeaddons. They have the same api group and as such require that both these CRDs are installed.

Reply:
just want to confirm. if `Addon` is installed. then `not (.Capabilities.APIVersions.Has "kubeaddons.mesosphere.io/v1alpha1")` will return `true`?

i'd assume it would return `false`.